lib/_emerge/DepPriority.py

```diff
@@ -1,11 +1,11 @@
-# Copyright 1999-2013 Gentoo Foundation
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 from _emerge.AbstractDepPriority import AbstractDepPriority
 
 
 class DepPriority(AbstractDepPriority):
-    __slots__ = ("satisfied", "optional", "ignored")
+    __slots__ = ("cross", "ignored", "optional", "satisfied")
```

Comment:
"cross" (as in cross-root) is fine with me, but maybe "altroot" or something would work if people are likely to get confused by it

Reply:
I've realized that this should also apply to the cross-prefix case where ROOT is /, if that matters for naming.

Reply:
thanks! cross it is then